libs/partners/together/langchain_together/chat_models.py

```diff
@@ -54,6 +55,14 @@ def _llm_type(self) -> str:
         """Return type of chat model."""
         return "together-chat"
 
+    def _get_ls_params(
+        self, stop: Optional[List[str]] = None, **kwargs: Any
+    ) -> LangSmithParams:
+        """Get the parameters used to invoke the model."""
+        params = super()._get_ls_params(stop=stop, **kwargs)
```

Comment:
Are we actually using `ls_model_name` in case of Azure? afaict, Azure needs only the deployment name, which does not necessarily have to mean model name

Reply:
should `AzureChatOpenAI` have `ls_model_name=self.deployment_name`?

Reply:
implemented that for now, lmk if it's not what you meant